File: Docs/CS/Data/Commands/MessageCommands.cs
namespace DevExpress.DevAV.Chat.Commands {
    public abstract class MessageCommand {
        public long MessageId {
            get;
        }
    }
    //
    public class DeleteMessage : MessageCommand {
        public DeleteMessage(long messageID)
            : base(messageID) {
        }
    }
    public class LikeMessage : MessageCommand {
        public LikeMessage(long messageID) 
            : base(messageID) {
        }
    }
}

File: Docs/VB/Data/Commands/MessageCommands.vb
Namespace DevExpress.DevAV.Chat.Commands
    Public MustInherit Class MessageCommand
        Public ReadOnly Property MessageId() As Long
    End Class
    '
    Public Class DeleteMessage
        Inherits MessageCommand

        Public Sub New(ByVal messageID As Long)
            MyBase.New(messageID)
        End Sub
    End Class
    Public Class LikeMessage
        Inherits MessageCommand

        Public Sub New(ByVal messageID As Long)
            MyBase.New(messageID)
        End Sub
    End Class
End Namespace